api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -1036,12 +1045,14 @@ select 0, 3112 as analysis_id,CAST(sm.concept_id as string) as stratum_1,
 CAST(o.observation_source_concept_id as string) as stratum_2,CAST(o.value_as_number as string) as stratum_4,
 '2' as stratum_5,COUNT(distinct p.PERSON_ID) as count_value,0 as source_count_value
 from \`${BQ_PROJECT}.${BQ_DATASET}.person\` p inner join \`${BQ_PROJECT}.${BQ_DATASET}.observation\` o on p.person_id = o.person_id
-join \`${BQ_PROJECT}.${BQ_DATASET}.concept_relationship\` cr on o.observation_source_concept_id = cr.concept_id_1
-join  \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_module\` sm on cr.concept_id_2=sm.concept_id
+join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_question_map\` sq
+On o.observation_source_concept_id=sq.question_concept_id
+join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_module\` sm on sq.survey_concept_id = sm.concept_id
 where (o.observation_source_concept_id > 0 and o.value_as_number is not null)
 and (o.observation_source_concept_id != 1585966)
```

Comment:
We can get rid of this now right since our map will have only questions we care about ?